Index: SDL2-2.0.12/configure.ac
===================================================================
--- SDL2-2.0.12.orig/configure.ac
+++ SDL2-2.0.12/configure.ac
@@ -3826,7 +3826,7 @@ AS_HELP_STRING([--enable-render-d3d], [e
         #  other spins, like x86-64, use a more standard "libstdc++.so.*"
         AC_CHECK_FILE("/boot/system/lib/libstdc++.r4.so", EXTRA_LDFLAGS="$EXTRA_LDFLAGS -lstdc++.r4", EXTRA_LDFLAGS="$EXTRA_LDFLAGS -lstdc++")
         ;;
-    arm*-apple-darwin*|*-ios-*)
+    a*-apple-darwin*|*-ios-*)
         ARCH=ios
 
         CheckVisibilityHidden
